nautobot_chatops/dispatchers/base.py

```diff
@@ -70,6 +70,7 @@ def platform_lookup(cls, item_type, item_name):
         """
         raise NotImplementedError
 
+    # pylint: disable=no-else-return
```

Comment:
This one could probably be fixed by simply removing the `else`, i.e.:

```python
    def static_url(self, path):
        """Construct an absolute URL for the given static file path, such as "nautobot/NautobotLogoSquare.png"."""
        static_path = str(static(path))
        if static_path.startswith("http"):
            return static_path
        return f"{self.context['request_scheme']}://{self.context['request_host']}{static_path}"
```

Reply:
done